app/scripts/store/dimStoreItem.directive.js

```diff
@@ -62,7 +62,7 @@
         '    <div class="img" dim-bungie-image-fallback="::vm.item.icon" ng-click="vm.clicked(vm.item, $event)">',
         '    <div ng-if="vm.item.quality" class="item-stat item-quality" ng-style="vm.item.quality.min | qualityColor">{{ vm.item.quality.min }}%</div>',
         '    <img class="element" ng-if=":: vm.item.dmg && vm.item.dmg !== \'kinetic\'" ng-src="/images/{{::vm.item.dmg}}.png"/>',
-        '    <div ng-if="vm.item.isNew" class="new_overlay_overflow">',
+        '    <div ng-if="vm.item.isNew && vm.settings.showNewAnimation" class="new_overlay_overflow">',
```

Comment:
Rather than adding another watcher to every item, we should just add a class at the base app that hides these elements.